signer/migrations/0003__create_tables.sql

```diff
@@ -13,14 +13,15 @@ CREATE TABLE sbtc_signer.bitcoin_blocks (
     block_hash BYTEA PRIMARY KEY,
     block_height BIGINT NOT NULL,
     parent_hash BYTEA NOT NULL,
-    confirms BYTEA[] NOT NULL,
     created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP NOT NULL
 );
 
 CREATE TABLE sbtc_signer.stacks_blocks (
     block_hash BYTEA PRIMARY KEY,
     block_height BIGINT NOT NULL,
     parent_hash BYTEA NOT NULL,
+    consensus_hash BYTEA NOT NULL,
```

Comment:
In the current implementation we don't actually need this stored -- it's enough to have it when building the stacks block model. Should we remove it?

Reply:
I'm inclined to remove it. If it turns out we actually need it down the line we can easily resurrect it. But I do not feel strongly one way or the other.

Reply:
Remove in [6802fa2](https://github.com/stacks-network/sbtc/pull/708/commits/6802fa27d2b2734a1b764930a380a64562dd6d4e)